storage_service/locations/api/resources.py

```diff
@@ -465,6 +491,34 @@ def dehydrate_misc_attributes(self, bundle):
         # Serialize JSONField as dict, not as repr of a dict
         return bundle.obj.misc_attributes
 
+    def hydrate_current_location(self, bundle):
```

Comment:
I think this should have a docstring to reduce the amount of tastypie knowledge that is required to understand this functionality. Also, I think the `LOGGER.debug` calls should be to `.info` and `.warning` in production, no?